tests/unittests/GraphOptzTest.cpp

```diff
@@ -273,16 +273,42 @@ TEST_F(GraphOptz, optimizeBatchNormAfterConvFP16) {
 
   EXPECT_EQ(F_->getNodes().size(), 3);
 
+  optimizedF_ = F_->clone(F_->getName().str() + "_optimized");
   ::glow::convertPlaceholdersToConstants(F_, bindings_, {});
-  ::glow::optimize(F_, CompilationMode::Infer);
-  EXPECT_EQ(F_->getNodes().size(), 2);
+  ::glow::convertPlaceholdersToConstants(optimizedF_, bindings_, {});
+  ::glow::optimize(optimizedF_, CompilationMode::Infer);
 
-  ASSERT_EQ(A->getNumUsers(), 1);
-  Node *newCV = A->getUsers().begin()->getUser();
-  EXPECT_TRUE(llvm::isa<ConvolutionNode>(newCV));
-  ASSERT_EQ(newCV->getNumUsers(), 1);
-  Node *save = newCV->getUsers().begin()->getUser();
-  EXPECT_TRUE(llvm::isa<SaveNode>(save));
+  EXPECT_EQ(optimizedF_->getNodes().size(), 2);
+
+  ASSERT_EQ(A->getNumUsers(), 2);
+
+  bool originalPathExists{false};
+  bool optimizedPathExists{false};
+  for (const auto &path : A->getUsers()) {
+    auto cv = path.getUser();
+    EXPECT_TRUE(llvm::isa<ConvolutionNode>(cv));
+    ASSERT_EQ(cv->getNumUsers(), 1);
+    auto next = cv->getUsers().begin()->getUser();
+    if (llvm::isa<SaveNode>(next)) {
+      EXPECT_FALSE(optimizedPathExists);
+      optimizedPathExists = true;
+    } else {
+      EXPECT_TRUE(llvm::isa<BatchNormalizationNode>(next));
+      ASSERT_EQ(next->getNumUsers(), 1);
+      auto origSave = next->getUsers().begin()->getUser();
+      EXPECT_TRUE(llvm::isa<SaveNode>(origSave));
+      EXPECT_FALSE(originalPathExists);
+      originalPathExists = true;
+    }
+  }
+
+  EXPECT_TRUE(originalPathExists);
+  EXPECT_TRUE(optimizedPathExists);
```

Comment:
You don't need to test the the original path exists. Just testing for the optimized path is fine.

Reply:
Got it!